qiskit_experiments/framework/base_experiment.py

```diff
@@ -192,8 +207,94 @@ def from_config(cls, config: Union[ExperimentConfig, Dict]) -> "BaseExperiment":
             ret.set_transpile_options(**config.transpile_options)
         if config.run_options:
             ret.set_run_options(**config.run_options)
+        if config.analysis:
+            ret.analysis = config.analysis.analysis()
         return ret
 
+    def save(self, service=None, backend=None):
+        """Save an experiment without running it."""
+        if not service:
+            raise QiskitError("A service must be provided to save the experiment.")
+        if not self.backend and not backend:
+            raise QiskitError("Backend must be set to save the experiment.")
+        exp_data = self._initialize_experiment_data(
+            service=service, backend=backend or self.backend
+        )
+        exp_data.save()
+        return exp_data
+
+    @classmethod
+    def load(
+        cls,
+        experiment_id: str,
+        service: Optional[IBMExperimentService] = None,
+        provider: Optional[Provider] = None,
+    ) -> "BaseExperiment":
+        """Load a saved experiment from a database service.
+
+        Args:
+            experiment_id: Experiment ID.
+            service: the database service.
+            provider: an IBMProvider required for loading the experiment data and
+                can be used to initialize the service. When using
+                :external+qiskit_ibm_runtime:doc:`qiskit-ibm-runtime <index>`,
+                this is the :class:`~qiskit_ibm_runtime.QiskitRuntimeService` and should
+                not be confused with the experiment database service
+                :meth:`qiskit_ibm_experiment.IBMExperimentService`.
+
+        Returns:
+            The reconstructed experiment.
+        Raises:
+            QiskitError: If the experiment could not be reconstructed.
+        """
+        if service is None:
+            if provider is None:
+                raise QiskitError(
+                    "Loading an experiment requires a valid Qiskit provider or experiment service."
+                )
+            service = ExperimentData.get_service_from_provider(provider)
+
+        data = service.experiment(experiment_id, json_decoder=ExperimentDecoder)
+
+        # Recreate artifacts
+        experiment_config_filename = "experiment_config.zip"
```

Comment:
Better store the name in one place to be used here and in `ExperimentData.__init__" (maybe in helpers.py to avoid cyclic dependency)